frontend/app/people/FormFields.tsx

```
@@ -43,7 +44,14 @@ export default function FormFields() {
                   <CommandList>
                     <CommandGroup>
                       {uniqueRoles.map((option) => (
-                        <CommandItem key={option} value={option} onSelect={(e) => field.onChange(e)}>
+                        <CommandItem
+                          key={option}
+                          value={option}
+                          onSelect={(e) => {
+                            field.onChange(e);
+                            setRolePopoverOpen;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)